Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -8273,10 +8273,11 @@ WebGPUEnabled:
 
 WebGPUHDREnabled:
   type: bool
-  status: internal
+  status: unstable
   category: dom
   humanReadableName: "WebGPU support for HDR"
   humanReadableDescription: "WebGPU High Dynamic Range through canvas configuration tone mapping mode"
+  condition: HAVE(HDR_SUPPORT)
```

Comment:
I think this is causing build failures on down levels, it seems fine to have the flag there, it will do nothing since WebGPU is disabled by default outside of Safari Technology Preview

Reply:
As discussed I've now removed that condition.
And I've also removed the `EnabledBySettings` against GPUCanvasConfiguration.toneMapping, so the configuration option will always be there (per spec). But we'll only accept the "extended" mode where both supported AND allowed by the feature flag.